tests/validators/test_table_schema.py

```diff
@@ -0,0 +1,291 @@
+"""Tests for the table_schema module."""
+
+from csvy.validators.table_schema import (
+    BooleanColumnValidator,
+    CommonValidator,
+    ConstraintsValidator,
+    DateTimeColumnValidator,
+    DefaultColumnValidator,
+    FormatDateTime,
+    FormatDefault,
+    FormatGeoJSON,
+    FormatGeoPoint,
+    FormatString,
+    GeoJSONColumnValidator,
+    GeopointColumnValidator,
+    IntegerColumnValidator,
+    NumberColumnValidator,
+    SchemaValidator,
+    StringColumnValidator,
+    TypeEnum,
+)
+
+
+def test_common_validator():
+    """Test the CommonValidator class."""
+    validator = CommonValidator(
+        **{
+            "name": "test_column",
+            "title": "Test Column",
+            "example": "example_value",
+            "description": "This is a test column.",
+            "constraints": {
+                "required": True,
+                "unique": True,
+            },
+        }
+    )
+    assert validator.name == "test_column"
+    assert validator.title == "Test Column"
+    assert validator.example == "example_value"
+    assert validator.description == "This is a test column."
+    assert validator.constraints.required is True
+    assert validator.constraints.unique is True
+
+    # Test model_dump
+    dumped = validator.model_dump()
+    assert dumped["name"] == "test_column"
+    assert dumped["title"] == "Test Column"
+    assert dumped["example"] == "example_value"
+    assert dumped["description"] == "This is a test column."
+    assert dumped["constraints"]["required"] is True
+    assert dumped["constraints"]["unique"] is True
```

Comment:
I think the way you've written these tests is v clear, but just to note that if `dump_model` is changed to add extra values, these tests won't catch it. Instead you *could* write:

```py
assert dumped == {"name": "test_column", # etc.
```

Same for the others.

Reply:
I'm not sure what you mean by "add extra values". In any case, what would be the problem with those extra values? As long as the ones that must be there, are there, all is good, right?